GUI/Main/MainInstall.cs

```diff
@@ -417,16 +417,26 @@ private void PostInstallMods(object sender, RunWorkerCompletedEventArgs e)
                     );
                 }
             }
-            else
+            else if (e.Error != null)
```

Comment:
Can you reverse this test and swap the then and else around? Always better to avoid a negative test for code readability.